dockers/docker-dhcp-relay/config.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
 
-sonic-cfggen -m /etc/sonic/minigraph.xml -y /etc/sonic/dhcp_relay.yml -t /usr/share/sonic/templates/isc-dhcp-relay.j2 > /etc/default/isc-dhcp-relay
+MINIGRAPH_HOSTNAME=`sonic-cfggen -m /etc/sonic/minigraph.xml -v "minigraph_hostname"`
+DEVICE_ROLE=`sonic-cfggen -m /etc/sonic/minigraph.xml -v "minigraph_devices['$MINIGRAPH_HOSTNAME']['type']"`
+
```

Comment:
you can directly use -v "minigraph_devices[minigraph_hostname]['type']", don't need to call it twice.